build.sh

```diff
@@ -131,7 +132,15 @@ if [ "${B_FS}" = "y" ] && [ -d  "${PREFIX_ROOTSKEL}" ]; then
 
 	mkdir -p "${PREFIX_ROOTFS}"
 	cp -a "${PREFIX_ROOTSKEL}/." "${PREFIX_ROOTFS}"
-	mkdir -p "$PREFIX_ROOTFS/"{dev,local,data,mnt,tmp,var}
+	mkdir -p "$PREFIX_ROOTFS/"{dev,etc,local,data,mnt,tmp,var,usr}
```

Comment:
side note: maybe this list should also be modifable by project config in the future (no need to change anything right now)